spec/spec_helper_acceptance.rb

```diff
@@ -1,22 +1,5 @@
-require 'beaker-rspec'
-require 'pry'
+require 'voxpupuli/acceptance/spec_helper_acceptance'
 
-hosts.each do |host|
-  # Install Puppet
-  on host, install_puppet
-end
-
-RSpec.configure do |c|
-  module_root = File.expand_path(File.join(File.dirname(__FILE__), '..'))
-
-  c.formatter = :documentation
-
-  # Configure all nodes in nodeset
-  c.before :suite do
-    # Install module
-    puppet_module_install(source: module_root, module_name: 'monit')
-    hosts.each do |host|
-      on host, puppet('module', 'install', 'puppetlabs-stdlib'), acceptable_exit_codes: [0, 1]
-    end
-  end
+configure_beaker do |host|
+  install_package(host, 'epel-release') if fact_on(host, 'os.family') == 'RedHat'
```

Comment:
Using this as a learning experience: you can also apply this natively with Puppet. In https://github.com/voxpupuli/voxpupuli-acceptance/?tab=readme-ov-file#per-host-configuration I've tried to document this

You can create `spec/setup_acceptance_node.pp` with content like:
```puppet
if $facts['os']['family'] {
  package { 'epel-release':
    ensure => installed,
  }
}
```

Then you can fully sync this file, which we [don't do by default](https://github.com/voxpupuli/modulesync_config/blob/9eb170bcdc0ef8e65795080901136cffcaa82081/config_defaults.yml#L116-L117).

To be clear, the current solution is also fine and valid. I just wanted to point this out after our conversation yesterday.

Reply:
ohhh, this one I very much like. That seems to be something we could tackle in all our modules spec tests to be able to start syncing the spec_helper_acceptance. I will keep that in mind when I tackle some of our other modules :)